packages/s/sfml/xmake.lua

```diff
@@ -3,18 +3,15 @@ package("sfml")
     set_homepage("https://www.sfml-dev.org")
     set_description("Simple and Fast Multimedia Library")
 
-    if is_plat("windows", "linux") then
+    -- Before 2.6.0 only x86 is supported for Mac
+    if is_plat("macosx") and is_arch("x64", "x86_64") then
+        set_urls("https://www.sfml-dev.org/files/SFML-$(version)-macOS-clang.tar.gz")
+        add_versions("2.5.1", "6af0f14fbd41dc038a00d7709f26fb66bb7ccdfe6187657ef0ef8cba578dcf14")
+    elseif is_plat("windows", "linux", "macosx") then
         set_urls("https://www.sfml-dev.org/files/SFML-$(version)-sources.zip")
         add_urls("https://github.com/SFML/SFML/releases/download/$(version)/SFML-$(version)-sources.zip")
         add_versions("2.5.1", "bf1e0643acb92369b24572b703473af60bac82caf5af61e77c063b779471bb7f")
-    elseif is_plat("macosx") then
-        if is_arch("x64", "x86_64") then
-            set_urls("https://www.sfml-dev.org/files/SFML-$(version)-macOS-clang.tar.gz")
-            add_versions("2.5.1", "6af0f14fbd41dc038a00d7709f26fb66bb7ccdfe6187657ef0ef8cba578dcf14")
-
-            add_configs("debug", {builtin = true, description = "Enable debug symbols.", default = false, type = "boolean", readonly = true})
-            add_configs("shared", {description = "Build shared library.", default = true, type = "boolean", readonly = true})
-        end
+        add_versions("2.6.0", "dc477fc7266641709046bd38628c909f5748bd2564b388cf6c750a9e20cdfef1")
     elseif is_plat("mingw") then
         if is_arch("x64", "x86_64") then
             set_urls("https://www.sfml-dev.org/files/SFML-$(version)-windows-gcc-7.3.0-mingw-64-bit.zip")
```

Comment:
@waruqi How do I pin a specific URL for a specific version?

i.e: 2.5.1 should point to `"https://www.sfml-dev.org/files/SFML-$(version)-windows-gcc-7.3.0-mingw-64-bit.zip"`
And 2.6.0 should point to `"https://www.sfml-dev.org/files/SFML-$(version)-windows-gcc-13.1.0-mingw-64-bit.zip"`

Reply:
see https://github.com/xmake-io/xmake-repo/blob/b5120abd24485ee46e6758185f05dd2626ca9b08/packages/s/sqlite3/xmake.lua#L6